### PR TITLE
PR template

### PR DESCRIPTION
**Pull request by EdNoepel**
## Description

Updated the PR template.  The old template required copy/paste mandates on contract size and gas consumption.  This template converts those to checkboxes.  Submitter can review CI output to confirm contract size and gas usage is appropriate.

## Purpose

Reduce amount of friction required to document a change.

## Impact

none

## Tasks

- [x] Changes to protocol contracts are covered by unit tests executed by CI.
- [x] Protocol contract size limits have not been exceeded.
- [x] Gas consumption for impacted transactions have been compared with the target branch, and nontrivial changes cited in the _Impact_ section above.
- [x] Scope labels have been assigned as appropriate.
- [x] Invariant tests have been manually executed as appropriate for the nature of the change.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,37 +1,24 @@
...
 
-<!---
-No need to add special tag
-src/ & non src/ changes you need the following (that apply):
--->
-# Description of change
-## High level
-* <DESCRIP_OF_CHANGE>
-  * <DESCRIP_OF_SUBCHANGE>
+Explain what was changed.  For example:
```
Comment (ith-harvey) [inline .github/pull_request_template.md]: Suggest hiding your examples text using `<!--` and `-->`

Comment (EdNoepel) [reply .github/pull_request_template.md]: Done in b542d2dbd362eacdc0a8c643099a34bb1f5a115c.

**Review by ith-harvey — APPROVED** [review]
LGTM

**Review by prat-gpt — APPROVED** [review]
LGTM

**Pull request merged**